_installation_collection/DE/Wie_installiere_ich_den_metasfresh_Stack_mit_Docker.md

```diff
@@ -63,6 +66,9 @@ Diese Anleitung beschreibt die Installation der folgenden Dienste gemäß [diese
 
    >**Hinweis:** Beim ersten Start kann es ein paar Minuten dauern, bis die Datenbank initialisiert und die Services verfügbar sind.
 
+1. (Optional) Das Docker Project wieder stoppen. <br>
```

Comment:
@n3w warum hast du diesen Punkt aufgenommen?

Reply:
Weil man den, gerade wenn man am Anfang mit der config bastelt, sicher braucht, oder gibt es da was besseres um die config neu zu laden?

Reply:
> (Optional) Das Docker Project wieder stoppen. <br>

verstehe. es geht dir darum den befehl zum runterfahren direkt mit zu dokumentieren,

Reply:
Hi @metasnw just a friendly reminder: for the sake of transparency, please write in english in the public repos :-)